rxandroid/src/main/java/rx/android/schedulers/HandlerScheduler.java

```diff
@@ -13,122 +13,29 @@
  */
 package rx.android.schedulers;
 
-import java.util.concurrent.TimeUnit;
-
-import rx.Scheduler;
-import rx.Subscription;
-import rx.android.plugins.RxAndroidPlugins;
-import rx.android.plugins.RxAndroidSchedulersHook;
-import rx.exceptions.OnErrorNotImplementedException;
-import rx.functions.Action0;
-import rx.plugins.RxJavaPlugins;
-import rx.subscriptions.Subscriptions;
 import android.os.Handler;
-import android.os.Message;
+import android.os.Looper;
+import rx.Scheduler;
 
-/** A {@link Scheduler} backed by a {@link Handler}. */
-public final class HandlerScheduler extends Scheduler {
-    /** Create a {@link Scheduler} which uses {@code handler} to execute actions. */
+/**
+ * A {@link Scheduler} backed by a {@link Handler}.
+ *
+ * @deprecated Use {@link AndroidSchedulers#from(Looper)}.
+ */
+@Deprecated
```

Comment:
Next is 1.2. I don't see a reason to do a major release for it and having
major versions out of sync with RxJava is going to be a mess.

On Mon, May 2, 2016, 9:14 AM Daniel Lew <notifications@github.com> wrote:

> In rxandroid/src/main/java/rx/android/schedulers/HandlerScheduler.java
> <https://github.com/ReactiveX/RxAndroid/pull/282#discussion_r61736328>:
>
> >
> > -/** A {@link Scheduler} backed by a {@link Handler}. */
> > -public final class HandlerScheduler extends Scheduler {
> > -    /** Create a {@link Scheduler} which uses {@code handler} to execute actions. */
> > +/**
> > + * A {@link Scheduler} backed by a {@link Handler}.
> > + *
> > + * @deprecated Use {@link AndroidSchedulers#from(Looper)}.
> > + */
> > +@Deprecated
>
> If we're going to another major revision for the next release (2.x), why
> not just remove it entirely?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ReactiveX/RxAndroid/pull/282/files/0bb9b4e63f5560ca238233f1c4d692d9bc8b5ba0#r61736328>
>